Dockerfile

```diff
@@ -77,6 +76,12 @@ COPY . "$BUILD_DIR"
 
 # Build guacamole-server from local source
 RUN yum -y install $BUILD_DEPENDENCIES         && \
+    curl -L https://codeload.github.com/LibVNC/libvncserver/tar.gz/LibVNCServer-0.9.11 | tar -zx && \
+    cd libvncserver-LibVNCServer-0.9.11 && \
+    ./autogen.sh && \
+    make install && \
+    cd .. && \
+    rm -fr libvncserver-LibVNCServer-0.9.11 && \
```

Comment:
Rather than embedding the build steps within the `Dockerfile`, they should instead be within a documented shell script. Besides being cleaner and easier to maintain, this removes the need to `cd ..` to restore the working directory.

See: https://github.com/apache/incubator-guacamole-server/blob/6236eb8f98ff96c495ae94cb4fe3279c202db465/src/guacd-docker/bin/build-guacd.sh